data/saas/config/mandrill_config.yml

```diff
@@ -0,0 +1,44 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Mandril SaaS Config
+  type: mandrill
+  description: A sample schema representing the Mandril connector for Fided
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: api_key
+
+  client_config:
+    protocol: https
+    host: <domain>
+
+  consent_requests:
+    opt_in:
+      - method: POST
```

Comment:
Still broader unknowns about how many individual requests about how many requests will be needed.  Here, I'm allowing a list of opt-in requests for different data uses and a list of opt out requests for different data uses.